media-video/obs-studio/files/com.obsproject.Studio.amf.desktop

```diff
@@ -0,0 +1,97 @@
+[Desktop Entry]
+Version=1.0
+Name=OBS Studio (AMF)
+GenericName=Streaming/Recording Software
+Comment=Free and Open Source Streaming/Recording Software
+Exec=env AMD_VULKAN_ICD=amdvlk-pro obs
```

Comment:
As we discovered `AMD_VULKAN_ICD` doesn't work, so here we either need to use `vk_pro` (thus we need to wait for the other PR to get merged) or explicitly do the VK_DRIVER_FILES override.

I think that Intel also does encoding through Vulkan, do you have an Intel iGPU to test ? Otherwise I think we probably should simply drop this PR :/